doc/build/orm/extensions/asyncio.rst

```diff
@@ -720,13 +732,21 @@ constructor::
 
     from asyncio import current_task
 
-    from sqlalchemy.ext.asyncio import async_sessionmaker
-    from sqlalchemy.ext.asyncio import async_scoped_session
-    from sqlalchemy.ext.asyncio import AsyncSession
-
-    async_session_factory = async_sessionmaker(some_async_engine, expire_on_commit=False)
-    AsyncScopedSession = async_scoped_session(async_session_factory, scopefunc=current_task)
+    from sqlalchemy.ext.asyncio import (
+        AsyncSession,
+        async_scoped_session,
+        async_sessionmaker,
+    )
 
+    async_session_factory = async_sessionmaker(
+        some_async_engine,
+        expire_on_commit=False,
+        class_=AsyncSession,
```

Comment:
I think we can remove `class_` here, since that's implied by the `async_sessionmaker`. 

should be removed also by the imports once updated

Reply:
yeah this should be removed